src/App.tsx

```diff
@@ -1,35 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.scss';
-import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title
+      .toLowerCase()
+      .includes(query.toLowerCase())
```

Comment:
```suggestion
      .includes(query.toLowerCase().trim())
```
<img width="521" alt="image" src="https://user-images.githubusercontent.com/55802630/184635998-e5afce9f-763e-4315-b4f6-682bcbcfb474.png">

(let's also fix the left padding 😉 )